lib/event_tracer/config.rb

```diff
@@ -9,5 +9,6 @@ class Config
     # TODO: switch to namespace in v1.0
     setting :dynamo_db_table_name, default: 'logs'
     setting :dynamo_db_client
+    setting :dynamo_db_queue_name, default: 'low'
```

Comment:
Should we use this field somewhere?

Reply:
You're right, I must have lost the change somehow.